opentelemetry-sdk/src/opentelemetry/sdk/_metrics/export/__init__.py

```diff
@@ -193,7 +197,7 @@ def _ticker(self) -> None:
         # one last collection below before shutting down completely
         self.collect()
 
-    def _receive_metrics(self, metrics: Iterable[Metric]) -> None:
+    def _receive_metrics(self, metrics: Iterable[Metric], **kwargs) -> None:
```

Comment:
Why do we need this for internal methods?

Reply:
Well, its a bit hard to tell if we would need them or not, so it would be safer to have them. :shrug:

Reply:
Right but internal methods have no interface guarantees, meaning they could change at any point in time.

Reply:
So there is no point of adding **kwargs right?

Reply:
Internal methods do have an interface guarantee if they are marked as `abstractmethod` for the user to implement them, as `_receive_metrics` is.

Reply:
+1 This one is "protected". We could also make it "public" alternatively, but the intention is that this template method is only called by the super class's final collect() method